MonoGame.Framework/Design/PointTypeConverter.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Text;
```

Comment:
Only include `using` statements for namespaces actually used.  Easiest way to do this in Visual Studio is right-click on the using block, select `Organize Usings` and then select `Remove and Sort`.